### ISSUE TITLE
Webpage not functional?

### ISSUE THREAD

**Issue by mickeN89** [msgpack/msgpack#306]
Hello, seems like the website is not working anymore?
the "Try!" link is not responding and the "Languages"/"API" field is empty

**Comment by tagomoris** [comment]
That's true... many errors are shown on the browser's console:
<img width="545" alt="Screen Shot 2021-04-07 at 17 26 27" src="https://user-images.githubusercontent.com/230654/113835019-7bb96b00-97c6-11eb-8614-6dd38781a7e5.png">

**Comment by zipzit** [comment]
How does one use this library?  Is this a dead project, or just a really dead website?  What languages are supported? Anybody have github links?  thx.

**Comment by markusschaber** [comment]
@zipzit This repository just contains the definition of the MessagePack format (describing the standard), it never provided an actual implementation.

The Repository list for msgpack contains some implemenations: https://github.com/msgpack

Also, you could do what the [crawler for the webpage](https://github.com/msgpack/website) does and search for the [tag on github](https://github.com/search?q=msgpack.org%5B).

Let's hope they will fix it soon.

**Comment by ericmutta** [comment]
It's 19th/June/2021 and the website is still broken!

If like me you are looking for the C# implementation the repo is here https://github.com/msgpack/msgpack-cli

**Comment by ats-org** [comment]
Yeah website is still broken, I think @frsyuki is the one that manages that site.

**Comment by janto** [comment]
For the last month there's been a PR waiting for review: https://github.com/msgpack/website/pull/44
Looks like mostly gem version updates.

**Comment by janto** [comment]
I mentioned @frsyuki about this on twitter https://twitter.com/janto_d/status/1445935112382734337?s=20 (could not DM). I think it's kinda important for msgpack to have a working website 😉

**Comment by vkaushal288-alivecor** [comment]
The website is still broken!

**Comment by janto** [comment]
@tagomoris do you have permissions to review and merge the relevant PR https://github.com/msgpack/website/pull/44?

@repeatedly seems to be the last person other than @frsyuki to have committed to the website's repo.

**Comment by tagomoris** [comment]
@janto I can't.

**Comment by ats-org** [comment]
Look like it's finally back up!

**Comment by AArnott** [comment]
The page is up, but the crawler is down. :(

**Comment by AArnott** [comment]
Since the msgpack.org site is dead, long live https://msgpack.io/ !
